concent_api/api-integration-force-get-task-result-test.py

```diff
@@ -1,197 +1,341 @@
-
-    cluster_url = command_line[1]
-    return cluster_url
+(PROVIDER_PRIVATE_KEY,  PROVIDER_PUBLIC_KEY)  = generate_ecc_key_pair()
+(REQUESTOR_PRIVATE_KEY, REQUESTOR_PUBLIC_KEY) = generate_ecc_key_pair()
 
 
-def upload_new_file_on_cluster(task_id = '0', part_id = '0', current_time = 0):
+def upload_new_file_on_cluster(task_id = '0', part_id = '0', time_offset = TOKEN_EXPIRATION_TIME):
+    (file_check_sum, file_content, file_size, headers)  = prepare_file_for_transfer(part_id, task_id, time_offset)
+    response                                            = upload_file(file_content, headers)
+    return (response.status_code, file_size, file_check_sum)
 
-    file_content    = task_id
-    file_size       = len(file_content)
-    file_check_sum  = 'sha1:' + hashlib.sha1(file_content.encode()).hexdigest()
-    file_path       = '{}/{}/result'.format(task_id, part_id)
 
+def prepare_file_for_transfer(part_id, task_id, time_offset):
+    file_content                                        = task_id
+    file_size = len(file_content)
+    file_check_sum                                      = 'sha1:' + hashlib.sha1(file_content.encode()).hexdigest()
+    file_path                                           = '{}/{}/result'.format(task_id, part_id)
     file_transfer_token = message.FileTransferToken()
-    file_transfer_token.token_expiration_deadline       = int(datetime.datetime.now().timestamp()) + 3600
+    file_transfer_token.subtask_id                      = "sub_" + task_id
+    file_transfer_token.token_expiration_deadline       = int(datetime.datetime.now().timestamp()) + time_offset
     file_transfer_token.storage_cluster_address         = STORAGE_CLUSTER_ADDRESS
     file_transfer_token.authorized_client_public_key    = CONCENT_PUBLIC_KEY
     file_transfer_token.operation                       = 'upload'
-
-    file_transfer_token.files                   = [message.FileTransferToken.FileInfo()]
-    file_transfer_token.files[0]['path']        = file_path
-    file_transfer_token.files[0]['checksum']    = file_check_sum
-    file_transfer_token.files[0]['size']        = file_size
-
-    upload_token    = shortcuts.dump(file_transfer_token, CONCENT_PRIVATE_KEY, CONCENT_PUBLIC_KEY)
-    encrypted_token = b64encode(upload_token).decode()
-
-    authorized_golem_transfer_token = 'Golem ' + encrypted_token
-
+    file_transfer_token.files                           = [message.FileTransferToken.FileInfo()]
+    file_transfer_token.files[0]['path']                = file_path
+    file_transfer_token.files[0]['checksum']            = file_check_sum  # prepare_hash(file_check_sum, wrong_hash)
+    file_transfer_token.files[0]['size']                = file_size
+    upload_token                                        = shortcuts.dump(file_transfer_token,
+                                                                         CONCENT_PRIVATE_KEY,
+                                                                         CONCENT_PUBLIC_KEY
```

Comment:
This script should not assume that it has access to Concent's private key. It's meant to run against a live cluster where it acts as a normal client without any special knowledge about server's secrets.

I see that the key was being used here already before your changes so it's your mistake but please fix it as a part of this task. It's necessary if we want to be able to run these tests against staging.